--- origsrc/svn/build.sh	2008-07-13 23:07:35.000000000 -0500
+++ src/svn/build.sh	2008-07-13 23:09:46.078125000 -0500
@@ -39,6 +39,6 @@
 # build tools
 cd tools
 
-gcc -o gtk-key-snooper $(pkg-config --cflags gtk+-2.0 libglade-2.0) $(pkg-config --libs gtk+-2.0) -Wl,-E $(pkg-config --libs libglade-2.0) gtk-key-snooper.c
+gcc -o gtk-key-snooper $(pkg-config --cflags gtk+-2.0 libglade-2.0) $(pkg-config --libs gtk+-2.0) gtk-key-snooper.c -Wl,-E $(pkg-config --libs libglade-2.0)
 
 cd ..
